### PR TITLE
chore(typos): fix typos

### PR DESCRIPTION
**Pull request by stavares843**
Fix typos across codebase.

### CODE REVIEW

File: build-review-apps.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ which constitutes the **"Review App"**.
   - [Setup](#setup)
   - [Workflow](#workflow)
   - [Create script](#create-script)
-  - [You're done! 🎉](#youre-done-)
+  - [You're done! 🎉](#your-done-)
```
Comment (LuchoTurtle) [inline build-review-apps.md:18]: Everything's awesome except this change. This will break the table of contents. May you revert this line? 😄

Thanks!

**Review by LuchoTurtle — CHANGES_REQUESTED** [review]
Thanks for fixing the typos! Just make the small change requested and it should be instantly mergeable 😄

**Comment by LuchoTurtle** [comment]
@nelsonic 10 tests are failing because of `AuthPlug.Token.generate_jwt!/1`. Was `auth` changed at any point? 👀

File: build-review-apps.md (hunk 1 above)
Comment (stavares843) [reply build-review-apps.md:18]: thanks! will update 🔨

Comment (stavares843) [inline build-review-apps.md:18]:
```suggestion
  - [You're done! 🎉](#youre-done-)
  - ```

File: build-review-apps.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ which constitutes the **"Review App"**.
   - [Workflow](#workflow)
   - [Create script](#create-script)
   - [You're done! 🎉](#youre-done-)
+  - ```
```
Comment (stavares843) [inline build-review-apps.md:19]:
```suggestion
  - 
```

File: build-review-apps.md (hunk 1 above)
Comment (stavares843) [reply build-review-apps.md]: done 🔨

**Comment by nelsonic** [comment]
@stavares843 thanks very much for proactively fixing these! 🥇 
@LuchoTurtle nothing has changed in `auth` or `auth_plug` all my work is on branches. ⏳ 
I don't understand why this is failing. 🤷‍♂️ 
I just merged https://github.com/dwyl/mvp/pull/330 and the build passes on `main`: 
https://github.com/dwyl/mvp/actions
image

https://github.com/dwyl/mvp/actions/runs/4352536248
image

So I say we should just _merge_ this and see if it `AuthPlug.Token.generate_jwt!/1` resolves itself. 
I suspect that it has something to do with @stavares843 creating the PR off a ***fork*** 
which means that her CI build doesn't have access to the `AUTH_API_KEY` ... 💭

**Review by LuchoTurtle — APPROVED** [review]

**Comment by LuchoTurtle** [comment]
@nelsonic I'm merging on your assumption being correct, then 👍

**Pull request merged**